src/dashboard/src/main/migrations/0010_dip_upload_store.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
This should be checked to make sure it matches #417, which is what was actually merged.